AssetManagementTestProject/TestSetup/NUnitWebTestSetup.cs

```diff
@@ -24,7 +23,7 @@ public void SetUp()
         LogoutPopup = new LogoutPopupPage();
         Asserter = new Asserter.Asserter();
         MenuBarLeft = new LeftMenuPage();
-        ChangePw1stTime = new ChangePassword1stTimePage();
+
     }
     [TearDown]
     public void TearDown()
```

Comment:
The method of a child class cannot have the same name as the parent class if you want to do some different actions
Change the name to WebBaseTearDown and WebBaseSetUp

Reply:
Done